rst2pdf/style-to-yaml.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+'''
+    Convert older RSON stylesheets to YAML format
+
+    Run the script with the filename to convert, it outputs to stdout
+'''
+
+import sys
+import json
+import yaml
+from rst2pdf.dumpstyle import fixstyle
+
+from rson import loads as rloads
+
+if __name__ == '__main__':
+    fname = sys.argv[1]
+    # read rson from a file
+    sstr = open(fname, 'rb').read()
+    style_data = fixstyle(rloads(sstr))
+    # output the style as json, then parse that
+    json_style = json.dumps(style_data)
+    reparsed_style = json.loads(json_style)
+
+    yaml_style = yaml.dump(reparsed_style)
+    print(yaml_style)
```

Comment:
It would be nice to be able to direct this to a file rather than having to do so via the terminal. Perhaps we could have a `--save` flag to dump to `'.'.join((os.path.splitext(fname)[0], 'yaml'))`. If you do this, don't forget to check that the file doesn't exist first and probably error out if so.

Reply:
I think this is needed especially now I have support for multiple files so easily on the input. It was too much for today but I'll take another look when I can

Reply:
I _think_ I've added that now but feedback always welcome. I mean, it does work! But might not be excellent.